### ISSUE TITLE
Batch norm during evaluation 

### ISSUE THREAD

**Issue by baytasin** [Haichao-Zhang/FeatureScatter#5]
Hi, thank you very much for sharing this repo. I had a question about the batch norm in wideresnet. I'm not very familiar with pytorch. Are you turning batch norm off while evaluating the robust model against attacks? I couldn't figure it out.
Thank you very much in advance.

**Comment by Haichao-Zhang** [comment]
The batch norm is still used in evaluation but the statistics are not further updated. The corresponding line of code is here:
https://github.com/Haichao-Zhang/FeatureScatter/blob/a9c0eda20d01332712d83717dd85f289a82ab87f/fs_eval.py#L160


Basically, after ```net.eval()``` is called, the running mean and variance statistics will be freezed and not changed further.
The current running mean and variance statistics will be used for computation, which is what we want in the evaluation phase.
Please let me know  if you have any further questions.

**Comment by baytasin** [comment]
Thank you very much for your fast response.
